lib/widgets/booking/booking_preview_panel_action_bar.dart

```diff
@@ -88,11 +88,9 @@ class _BookingPreviewEditIconButton extends StatelessWidget {
     final editedBooking = await showDialog<Booking>(
       context: context,
       builder: (context) => BookingDialog(
-        booking: (booking.recurringBookingId == null
-            ? booking
-            : cabinManager
-                .cabinFromId(cabin.id)
-                .recurringBookingFromId(booking.recurringBookingId!))
+        booking: (booking.recurringBooking != null
+            ? booking.recurringBooking!
+            : booking)
```

Comment:
This change introduced an undesired behavior after popping a changed booking from the form dialog. It is addressed in #167.